pages/apply/expect.tsx

```diff
@@ -7,27 +7,90 @@ import Layout from '../../components/layout/resident-layout';
 import DeleteLink from '../../components/delete-link';
 import Announcement from '../../components/announcement';
 import { signOut } from '../../lib/store/cognitoUser';
+import { Applicant } from '../../domain/HousingApi';
+import { CALCULATE_BEDROOMS } from '../../lib/utils/bedroomCalculator';
+import { useState, useEffect } from 'react';
 
 const WhatToExpect = (): JSX.Element => {
   const router = useRouter();
   const dispatch = useAppDispatch();
 
+  const [CalculatedBedrooms, setCalculatedBedrooms] = useState(0);
+  const [waitingTime, setWaitingTime] = useState(0);
+
   const onCancel = () => {
     dispatch(signOut());
     router.push('/');
   };
 
+  const getAge = (date: string) => {
+    let today = new Date();
+    let birthDate = new Date(date);
+    let age = today.getFullYear() - birthDate.getFullYear();
+    let m = today.getMonth() - birthDate.getMonth();
+    if (m < 0 || (m === 0 && today.getDate() < birthDate.getDate())) {
+      age--;
+    }
+    return age;
+  };
+
+  const getGender = (gender: string) => {
```

Comment:
you don't need to do this. The calculate bedrooms function doesn't process strings like `male` and `female`. It's just looking to see if people have the same or a different gender. You can use anything you like - `M`, `male`, `boy` - it doesn't care as long as you're consistent.